sidekiq-superworker.gemspec

```diff
@@ -4,19 +4,16 @@ Gem::Specification.new do |s|
   s.authors       = ['Tom Benner']
   s.email         = ['tombenner@gmail.com']
   s.description = s.summary = %q{Define dependency graphs of Sidekiq jobs}
-  s.homepage      = 'https://github.com/socialpandas/sidekiq-superworker'
+  s.homepage      = 'https://github.com/Invoca/sidekiq-superworker'
 
   s.files         = Dir['{lib}/**/*'] + ['MIT-LICENSE', 'Rakefile', 'README.md']
   s.name          = 'sidekiq-superworker'
   s.require_paths = ['lib']
   s.version       = Sidekiq::Superworker::VERSION
   s.license       = 'MIT'
+  s.metadata["allowed_push_host"] = "https://gem.fury.io/invoca"
```

Comment:
I didn't want to deal with changing the gemname so I just used a new invoca source... Maybe we should just change the name 🤷

Reply:
Ideally we can push this back upstream, so I would prefer not to update the name.